src/config/nfs.md

```diff
@@ -0,0 +1,43 @@
+# NFS
+
+## Mounting an NFS Share
+
+To mount an NFS share, start by installing the `nfs-utils` package:
+
+```
+# xbps-install nfs-utils
+```
+
+Before you mount an NFS share, you'll need to [enable the statd and procbind
+services](services/index.md#enabling-services):
+
+(If your server supports `nfs4`, you do not need statd and you can replace `nfs`
+with `nfs4` below)
+
+```
+# sudo ln -s /etc/sv/rpcbind /var/service
+# sudo ln -s /etc/sv/statd /var/service
+```
+
+Now you're ready to mount a directory:
+
+```
+# mount -t nfs <host:/path/to/dir> /mnt/nfs/share
+```
+
+For example, if you wanted to connect `/volume` on a server at IP Address
+`192.168.1.99` to a directory `/mnt/volume` on your local system, you could do:
+
+```
+# mkdir /mnt/volume
+# mount -t nfs 192.168.1.99:/volume /mnt/volume
+```
+
+To have the directory mounted again when the system reboots, add an entry to
+`/etc/fstab`:
+
+```
+# echo "192.168.1.99:/volume /mnt/volume nfs rw,hard,intr 0 0" >> /etc/fstab
+```
```

Comment:
I don't like having a command like this... You can instead show a simple example line that could be in fstab.

Reply:
agreed. Also I discovered my nfs share didn't get mounted on boot - is there some service I need to enable to mount that or something?

Also void failed to shutdown, it blocked on `unmounting filesystems, disabling swap` - any ideas about that?

Reply:
https://github.com/void-linux/void-runit/issues/16 appears to be the issue I'm hitting - it doesn't seem that there's a real resolution to it? There are workarounds but they will get overwritten on system updates?